### DISCUSSION TITLE
Botton Labels

### DISCUSSION THREAD

**Discussion by seleneflemming** [JakubAndrysek/PySpaceMouse#21] [General]
Hello!
the 3dConnexion Spacemouse Enterprise has an LCD display which shows the current function of each button. Is there a way to customize this in pyspacemouse?
thanks!
Selene

**Comment by JakubAndrysek** [comment]
Hi, PySpaceMouse library currently supports only basic processing packets on HID USB. 
After a quick research, I found the https://github.com/TheHoodedFoot/SpaceLCD library, which separately adds support to control your display (Linux only).

I didn't have this type of mouse, so I'm not planning on adding display support anytime soon. But if you want, PR are welcome 😇.

Somewhere in the code is the example of requesting to turn on/off the mouse ring LED.

Have a nice day, Jacob.

**Comment by seleneflemming** [comment]
Fantastic thank you!!

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
